python-package/lightgbm/sklearn.py

```diff
@@ -919,6 +918,7 @@ def predict_proba(self, X, raw_score=False, start_iteration=0, num_iteration=Non
         """Docstring is set after definition, using a template."""
         result = super().predict(X, raw_score, start_iteration, num_iteration, pred_leaf, pred_contrib, **kwargs)
         if callable(self._objective) and not (raw_score or pred_leaf or pred_contrib):
+            new_line = "\n"
```

Comment:
I should have included this in my suggestion, sorry. I think this can be removed as well.

```suggestion
```